Dockerfile

```diff
@@ -60,6 +60,9 @@ RUN pip install ocrd
 # install mets-mods2tei and page-to-alto
 RUN pip install mets-mods2tei
 RUN pip install ocrd-page-to-alto
+# install mongosh
+RUN wget https://downloads.mongodb.com/compass/mongodb-mongosh_1.10.1_amd64.deb
```

Comment:
Does the `mongosh` version even match the MongoDB version if it is not configurable externaly?

Reply:
oh, good question!

We simply use `mongo:latest` image in the Monitor, and it says it's v6.0.6. I don't know how the versioning of the mongosh relates to that. I guess a CLI must support multiple versions anyway...